### PR TITLE
Msw 114005389 send reminder sms

### PR DESCRIPTION
**Pull request by michaelwehrley**

### CODE REVIEW

File: app/models/concerns/practice_reminder.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    def navigator_context_url
+      @navigator_context_url ||= ThinkFeelDoEngine::Engine
+                                 .routes
+                                 .url_helpers
+                                 .navigator_context_url(
+                                   context_name: tool_title)
+    end
+
+    def notify_sentry(message)
+      return unless defined? Raven
```
Comment (ericcf) [inline app/models/concerns/practice_reminder.rb]: Do you want to return an empty string here just like line 45?

**Comment by ericcf** [comment]
Other than my minor comment, looks great!

File: app/models/concerns/practice_reminder.rb (hunk 1 above)
Comment (michaelwehrley) [inline app/models/concerns/practice_reminder.rb]: Instead, I now have `tool_title` return an empty string.  I think that makes more sense anyway.  Thanks!

**Comment by michaelwehrley** [comment]
Merging commit.  Any other updates, I can make later.

**Pull request merged**